.asf.yaml

```diff
@@ -26,11 +26,16 @@ github:
     - universal-resource-scheduler
     - apache-yarn
     - kubernetes
+  notifications:
+    commits:      isseus@yunikorn.apache.org
```

Comment:
Is issues@yunikorn.apache.org -> isseus@yunikorn.apache.org?

Reply:
fixed on commit